src/client/query.rs

```diff
@@ -475,3 +475,27 @@ impl Drop for QueryResultSet {
         }
     }
 }
+
+pub struct ObTableAggregationResult {
+    /// use hash map to record the result of aggregation
+    aggregation_result: HashMap<String, Value>,
+}
+
+impl ObTableAggregationResult {
+    pub(crate) fn new() -> Self {
+        Self {
+            aggregation_result: HashMap::new(),
+        }
+    }
+
+    /// use query result to init the hash map
+    pub async fn init(mut self, mut query_result_set: QueryResultSet) -> Self {
+        self.aggregation_result = query_result_set.next().await.unwrap().unwrap();
```

Comment:
I guess we should remove unwarp() to avoid panic.
```suggestion
    if let Some(Some(aggregation_result)) = query_result_set.next().await {
        // dealing with success
        self.aggregation_result = aggregation_result;
    } else {
        // empty result or error
    }
```
or
```suggestion
    match query_result_set.next().await {
        Some(Some(aggregation_result)) => {
            // dealing with success
            self.aggregation_result = aggregation_result;
        },
        _ => {
            // empty result or error
        }
    }

Reply:
remove aggregation result